### PR TITLE
improve ESLint configuration

### PR DESCRIPTION
**Pull request by Ma27**
### Overview

- [x] improve eslint configuration (removed conflicts with the `recommended` preset, renamed deprecated rules)
- [x] fixed problem with eslint and static class properties using `babel-eslint`
- [x] fix all remaining styling things
- [x] add `eslint-pluginn-arca` with a plugin for import alignment

### CODE REVIEW

**Comment by Ma27** [comment]
note: we'll wait here until #357 is merged

**Review by benbieler — APPROVED** [review]
Well - the plugin seems to be working

**Comment by Ma27** [comment]
ok, thanks for the approval. Will merge tonight right after your PR :)

**Pull request merged**
